### ISSUE TITLE
Official Browsh .deb file does not install man page

### ISSUE THREAD

**Issue by ChaiTRex** [browsh-org/browsh#314]
I installed Browsh onto Ubuntu 18.04 using the instructions on https://www.brow.sh/docs/installation/. No man pages are installed:

    $ man browsh
    No manual entry for browsh
    See 'man 7 undocumented' for help when manual pages are not available.

**Comment by tombh** [comment]
I don't think we even have any man pages included at the moment, so that's the most likely explanation!
